lib/config.js

```diff
@@ -7,7 +7,10 @@ var config = require('bedrock').config;
 var path = require('path');
 require('bedrock-validation');
 
-config['key-http'] = {};
+// var basePath = '/keys';
+// config['key-http'] = {};
+// config['key-http'].routes = {};
+// config['key-http'].routes.basePath = basePath;
```

Comment:
Sounds like we should make `basePath` here a computed config variable based on bedrock-key's config var. @davidlehn ?